app/View/Composers/MainMenuComposer.php

```diff
@@ -120,8 +120,12 @@ private function addMembersMenu(Builder $menu)
                     $usersMenu->add('View all users', route('user.index'));
                     $usersMenu->add('Create a new user', route('user.create'));
                 }
-                
+
+                //Quotes Board
                 $membersMenu->add('Quotes Board', route('quotes.view'));
+
+                //Wiki
```

Comment:
sorry, gonna be "that guy"
```suggestion
                // Quotes Board
                $membersMenu->add('Quotes Board', route('quotes.view'));

                // Wiki
```

Reply:
Well spotted! Good to have it consistent.